cli.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Sets up the CLI command early in the WordPress load process.
+ *
+ * This is necessary to setup the environment to perform runtime checks.
+ *
+ * @package plugin-check
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
+	return;
+}
+
+// Check if the plugin autoloading is set up.
+if ( ! class_exists( 'WordPress\Plugin_Check\CLI\Plugin_Check_Command' ) ) {
+	// Check the autoload file exists.
+	if ( ! file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
+		WP_CLI::error( 'Plugin Check autoloaded not found.' );
+		return;
+	}
+
+	// Load the Composer autoloader.
+	require_once __DIR__ . '/vendor/autoload.php';
+}
+
+if ( ! isset( $context ) ) {
+	$context = new WordPress\Plugin_Check\Plugin_Context( __DIR__ . '/plugin-check.php' );
```

Comment:
Question: is there a reason why these are not imported at the top of the file this?

```
use WordPress\Plugin_Check\Plugin_Context;
```